### DISCUSSION TITLE
Renaming a label doesn't update the view using the label as filter

### DISCUSSION THREAD

**Discussion by jorge-campo** [community/community#17492] [Projects and Issues]
This is feedback for the new GitHub Projects beta.

**Actions**

- Create a new view in Project
- Filter the view using one of your labels (filtered view).
- Go to the repository issues and rename the label to something else.

**Result**

- The label used as a filter maintains the original name.
- The filtered view is empty.

**Expected result**

- The filter updates using the new label name.
- The filtered view shows the issues for that label.

**Comment by mvaldi** [comment]
I have the same issue, +1
